components/footer/index.css

```diff
@@ -0,0 +1,85 @@
+/*************************************************************************
+* ADOBE CONFIDENTIAL
+* ___________________
+*
+* Copyright 2019 Adobe
+* All Rights Reserved.
+*
+* NOTICE: All information contained herein is, and remains
+* the property of Adobe and its suppliers, if any. The intellectual
+* and technical concepts contained herein are proprietary to Adobe
+* and its suppliers and are protected by all applicable intellectual
+* property laws, including trade secret and copyright laws.
+* Dissemination of this information or reproduction of this material
+* is strictly forbidden unless prior written permission is obtained
+* from Adobe.
+**************************************************************************/
+
+
+:root {
+  --spectrum-footer-height : var(--spectrum-global-dimension-size-500);
+  --spectrum-footer-text-size : var(--spectrum-global-dimension-font-size-75);
+
+  --spectrum-footer-item-gap : 32px;
+  --spectrum-footer-row-gap-mobile : 48px;
+}
+
+.spectrum-Footer {
+  position: relative;
+  width: 100%;
+
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  min-height: var(--spectrum-footer-height);
+
+  font-size: var(--spectrum-footer-text-size);
+
+  .spectrum-Footer-divider {
+    width: 100%;
```

Comment:
Uhg rule and it's need for widths and heights again...

Reply:
do i need to do anything here? or just a general complaint?

Reply:
Nope, just complaining.